tests/system_tests/test_exceptions.py

```diff
@@ -258,22 +258,28 @@ def test_breaking_into_handled_exceptions(self):
             DebugInfo(filename=filename, cwd=cwd))
 
     def test_breaking_into_uncaught_exceptions(self):
+        DebugSessionConnection.VERBOSE = True
         filename = TEST_FILES.resolve('unhandled_exceptions_launch.py')
```

Comment:
This is something that should be done with a context manager (`with`)

Reply:
I won’t be using that. I added that temporarily to find a issue I was running into on Travis.